kf/nfconfig_test.go

```diff
@@ -261,7 +262,7 @@ func TestNFConfigs_Deploy(t *testing.T) {
 			wantErr: true,
 		},
 		{
-			name: "ValidHostNameValidIfaceName",
+			name: "ValidhostNameValidIfaceName",
 			fields: fields{
 				hostName:       machineHostname,
 				hostInterfaces: map[string]bool{"enp0s3": true},
```

Comment:
```suggestion
				hostInterfaces: map[string]bool{"fakeif0": true},
```
for consistency with line 488, 523, etc.